.eslintrc.json

```diff
@@ -21,26 +21,11 @@
     "no-param-reassign": 0,
     "comma-dangle": 0,
     "curly": ["error", "multi-line"],
-    "import/no-unresolved": [2, { "commonjs": true }],
-    "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
-    "valid-jsdoc": [
-      "error",
-      {
-        "requireReturn": true,
-        "requireReturnType": true,
-        "requireParamDescription": false,
-        "requireReturnDescription": true
-      }
-    ],
-    "require-jsdoc": [
```

Comment:
why did you remove the `require-jsdoc` and the above settings? Remember it was specified on eslint setup [task](https://www.pivotaltracker.com/story/show/167733632)

Reply:
> why did you remove the `require-jsdoc` and the above settings? Remember it was specified on eslint setup [task](https://www.pivotaltracker.com/story/show/167733632)

Sorry @emmanuelnwankwo It must have been an oversight